webui/react/src/components/TableBatch.tsx

```diff
@@ -14,24 +22,52 @@ const defaultProps = {
   selectedRowCount: 0,
 };
 
-const TableBatch: React.FC<Props> = (props: PropsWithChildren<Props>) => {
+const TableBatch: React.FC<Props> = ({
+  actions,
+  selectedRowCount,
+  onAction,
+  onClear,
+}: PropsWithChildren<Props>) => {
+  const [ action, setAction ] = useState<string>();
   const classes = [ css.base ];
-  const selectedRowCount = props.selectedRowCount || 0;
+  const selectCount = selectedRowCount || 0;
+
+  const message = `Apply batch operations to ${selectCount}`+
+    ` item${selectCount === 1 ? '' : 's'}`;
 
-  const message = `Apply batch operations to ${selectedRowCount}`+
-    ` item${selectedRowCount === 1 ? '' : 's'}`;
+  if (selectCount > 0) classes.push(css.show);
 
-  if (selectedRowCount > 0) classes.push(css.show);
+  const handleAction = useCallback((action?: string) => {
+    /*
+     * This succession setting of action to an empty string
+     * followed by `undefined` is required to guarantee clearing
+     * out of the selection value. Using a state `value` prop and
+     * setting the state to `undefined` did not work.
+     */
+    setAction('');
+    setTimeout(() => setAction(undefined), 100);
```

Comment:
This is really weird but I believe you that it didn't work otherwise

Reply:
crazy right? really open to ideas here because after trying a bunch of things, this was the most reliable solution that I found...